application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -223,9 +229,11 @@ private static ForumTag getTagOrDefault(List<ForumTag> tagsFoundOnForum,
     }
 
     private MessageEmbed makeEmbedForPost(User originalUser, String originalMessage) {
+        ImageProxy avatarOrDefault = originalUser.getEffectiveAvatar();
```

Comment:
could u do a quick full-text-search of `getAvatar()`? i suspect that bug might be elsewhere as well, for example in `/audit`

Reply:
https://github.com/search?q=repo%3ATogether-Java%2FTJ-Bot%20getAvatar&type=code

Reply:
ty squid, @Zabuzard  should i update em all?

Reply:
that would be cool, thanks